### DISCUSSION TITLE
IPF <101> & <111> Texture Dominance Issue

### DISCUSSION THREAD

**Discussion by B-steg** [mtex-toolbox/mtex#1730] [Ask Anything]
I work on some LPBF Ni superalloys and from my experience and XRD results, I generally get a highly textured <100> crystallographic texture coming out of the XY plane (parallel build direction). But recently I have been getting <101> and <111> texture on this plane. Its odd because I have done at this point >100 scans at this point and never have had this issue until now. Has anyone experienced this? Any suggestions would be appreciated.

<img width="299" alt="101-111 EBSD-IPF" src="https://user-images.githubusercontent.com/105690159/216835327-7e9faca8-1657-410e-97e8-4233e18dff5a.png">

**Comment by kilir** [comment]
Hi,
did you change in any way how you import data into mtex compared to previously?
Cheers,
Rüdiger
